llama_stack/apis/memory/client.py

```diff
@@ -42,7 +42,11 @@ async def get_memory_bank(self, bank_id: str) -> Optional[MemoryBank]:
                 params={
                     "bank_id": bank_id,
                 },
-                headers={"Content-Type": "application/json"},
+                headers={
```

Comment:
can we revert these changes from here, since this is a pretty generic client -- we will figure out how to get a bunch of these examples situated.

Reply:
done